docs/secure-connections.md

```diff
@@ -24,14 +24,14 @@ See our [Sauce Connect Proxy Security Overview](https://saucelabs.com/resources/
   <div className="box-wrapper" markdown="1">
     <div className="box box1 card">
       <div className="container">
-      <h3><a href="/secure-connections/sauce-connect">Using Sauce Connect Proxy</a></h3>
-      <p>Review the Sauce Connect Proxy system and network requirements, and then install the Sauce Connect Proxy client.</p>
+      <h3><a href="/secure-connections/sauce-connect">Sauce Connect Proxy Overview</a></h3>
+      <p>Learn about Sauce Connect Proxy system requirements and how to install the client.</p>
```

Comment:
```suggestion
      <p>See the Sauce Connect Proxy system requirements and learn how to install the client.</p>
```
^The description for the overview doesn't seem to align with the content in the linked topic.